Views/Lombiq.Privacy.ConsentBanner.cshtml

```diff
@@ -1,37 +1,40 @@
 @using Microsoft.AspNetCore.Http.Features;
 @using Lombiq.Privacy.Controllers;
 
+@inject Microsoft.AspNetCore.Antiforgery.IAntiforgery AntiForgery
+
 <style asp-name="@ResourceNames.ConsentBanner"></style>
 <script asp-name="jQuery"></script>
 
 @{
-    var bannerId = "privacy-consent-banner";
-    var acceptButtonId = "privacy-consent-accept-button";
-    var deniedButtonId = "privacy-consent-denied-button";
+    const string BannerId = "privacy-consent-banner";
+    const string AcceptButtonId = "privacy-consent-accept-button";
+    const string DeniedButtonId = "privacy-consent-denied-button";
```

Comment:
Let's use `camelCase` here, as is the case in 25 out of 27 occurrences in the OSOCE solution:

![image](https://user-images.githubusercontent.com/164360/165749551-e90a76fc-9d89-4bdb-8202-c06e5020d384.png)

Reply:
Ok, so more occurrence means better practice

Reply:
It should be like that. And we try to consolidate outliers so that wherever you look, you'll the correct version. In that spirit, could you please adjust the two occurrences at the bottom? Thanks 😃

Reply:
You can find `@inject`-ed varible names in `PascalCase` and `camelCase` too.
If you want a complete refactoring of OSOE, please make a new issue for that, or be my guest.

Reply:
I've created https://lombiq.atlassian.net/browse/OSOE-109 as a follow-up.